srcpkgs/avr-binutils/template

```diff
@@ -3,43 +3,21 @@ pkgname=avr-binutils
 version=2.32
 revision=1
 wrksrc="binutils-$version"
+build_style=gnu-configure
+configure_args="--disable-werror --enable-shared --enable-gold
+ --enable-plugins --enable-install-libbbfd --build=${XBPS-TRIPLET}
```

Comment:
`${XBPS_TRIPLET}` note the underscore, not dash.

Reply:
Woops typo :)